 <!DOCTYPE html>
<html lang="en-US">
<head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="css/styles.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="icon" type="image/png" href="imgs/favicon.png" />
    <title>Thomas Dorfer | About me</title>
</head>
<body>
  <ul class="nav" id="myTopnav">
    <a href="#top" class="active">About Me</a>
    <a href="#mywork" class="active">My Work</a>
    <a href="#protlearn" class="active">ProtLearn</a>
    <a href="#nlprot" class="active">NLProt</a>
    <a href="#technicalwriting" class="active">Technical Writing</a>
    <a href="javascript:void(0);" class="icon" onclick="responsiveness()">
      <i class="fa fa-bars"></i>
    </a>
  </ul>
    <div class="column-header">
         <h1 id="header">Thomas Dorfer</h1>
         <h3 style="color: #3c4043; text-align: center; font-size: 14px; font-weight: 100; padding: 0 0 85px 0">DATA SCIENTIST | MOLCURE Inc.</h3>
    </div> 
    <div class="content">
        <div class="left"></div>
        <div class="midcolumn">
            <div class="card" style="margin-top: -100px; margin-bottom: 200px;">
                <br>
                <div style="position: absolute; left: 50%;">
                  <div style="position: relative; left: -50%;">
                    <img class="center" src="imgs/rsz_img_0939.jpg" alt="Profile Picture" style="width:235px; height: 250px; box-shadow: 0px 8px 10px 0px rgba(0,0,0,0.2); border-radius: 50%;">
                  </div>
                </div> 
            </div>
            <div class="card">
              <div class="middle">
                <button class="button github"><a href="https://github.com/tadorfer" target="_blank">GitHub</a></button>
                <button class="button sourcerer"><a href="https://sourcerer.io/tadorfer" target="_blank">Sourcerer</a></button>
                <button class="button linkedin"><a href="https://www.linkedin.com/in/thomasdorfer/" target="_blank">LinkedIn</a></button>
                <button class="button twitter"><a href="https://twitter.com/ThomasADorfer" target="_blank">Twitter</a></button>
            </div>
                <p id="para">I'm a data scientist at Molcure Inc. in Tokyo, Japan, with training in neuroscience. My primary interests lie in machine learning and protein modeling, particularly 
                  the use of natural language processing to decipher the hidden grammar of protein sequences.
                </p>
            </div> 
            <div class="card">   
              <h2 style="font-family: arial;"><span id="mywork"></span>My Work</h3>
              <h3 style="margin-top: 40px; margin-bottom: 30px;">Conference Papers</h3>
              <p id="para"><b>Dorfer TA</b>, Robinson CN, Cocchi L, Mattingley JB, Sale MV, Zalesky A, Gollo LL.
                          Whole-brain network states predict behavioral responses to transcranial magnetic stimulation.
                          <i>Imaging@Brisbane</i>, Brisbane, August 2018. <button class="button-inline pdf"><a href="papers/ImagingBrisbane_2018.pdf" target="_blank">PDF</a></button></p>
              <p id="para">Pater MRA, <b>Dorfer TA</b>, Gschwind L, Coynel D, Papassotiropoulos A, de Quervain DJ, Luksys G.
                          Predicting Human Memory Performance through Multi-Voxel Pattern Analysis.
                          <i>Federation of European Neuroscience Societies</i>, Berlin, July 2018. <button class="button-inline pdf"><a href="papers/FENS_2018.pdf" target="_blank">PDF</a></button></p>
              <p id="para"><b>Dorfer TA</b>, Roberts JA, Breakspear M, Gollo LL.
                          Slow oscillatory brain activity renders convolution with the hemodynamic response function redundant.
                          <i>Organization for Human Brain Mapping</i>, Singapore, June 2018. <button class="button-inline pdf"><a href="papers/OHBM_2018_1.pdf" target="_blank">PDF</a></button></p>
              <p id="para"><b>Dorfer TA</b>, Pater M, Gschwind L, Papassotiropoulos A, de Quervain DJ, Luksys G.
                          FMRI-based prediction models for free recall, recognition memory, emotional valences, arousal, and memorability of pictures.
                          <i>Organization for Human Brain Mapping</i>, Singapore, June 2018. <button class="button-inline pdf"><a href="papers/OHBM_2018_2.pdf" target="_blank">PDF</a></button></p>
              <p id="para">Gollo LL, Cocchi L, Hearne L, <b>Dorfer TA</b>, Roberts J, Breakspear M.
                          Can we predict the intensity of the effects of brain stimulation?
                          <i>Organization for Human Brain Mapping</i>, Singapore, June 2018.</p>
          </div>
          <div class="card">
            <h2 style="font-family: arial;"><span id="protlearn"></span>ProtLearn</h2>
            <p id="para">ProtLearn is a feature extraction tool for protein sequences. It is a freely available Python package that allows the user to efficiently extract amino acid sequence features from proteins and peptides, which can then be used for a variety of downstream machine learning tasks. </p><br><br>
            <p style="text-align: center;"><button class="button-inline pdf"><a href="https://github.com/tadorfer/protlearn" target="_blank">Source code</a></button> <button class="button-inline pdf"><a href="https://protlearn.readthedocs.io/en/latest/index.html" target="_blank">Documentation</a></button>
            </p><br>
              <div class="container">
                  <img class="center" src="imgs/protlearn_summary.png" alt="ProtLearn Overview" style="width:624px; height: 430px;">
              </div> 
          </div>
          <div class="card">
            <h2 style="font-family: arial;"><span id="nlprot"></span>Natural Language Processing for Proteins (NLProt)</h2>
            <p id="para">The application of Natural Language Processing (NLP) to protein sequence prediction has recently gained traction in the fields of machine learning and computational biology. 
                This was primarily fueled by the recent advances in deep learning and language models such as Google's BERT and its successors RoBERTa and ALBERT. 
                The aim of this site is to provide a comprehensive and chronologically ordered list of the recently published literature in this area. If you have come across relevant work that should be added to this list, please feel free to make a pull request or open an issue <a href="https://github.com/tadorfer/NLProt" target="_blank" style="color:gray; text-decoration: none;">here</a>. 
            </p>
        </div> 
        <div class="card">
            <div class="container">
                <iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://plotly.com/~tadorfer/6.embed?showlink=false&modebar=false" height="550" width="100%"></iframe>
                <p style="text-align: center; font-size: 15px; color: gray; padding-top: 0px">Papers (blobs) arranged by similarity and date (the more recent, the bigger).</p>
            </div>
        </div>
        <br>
        <div class="middle">
          <button class="button_papers" onclick="showPapers()" id="myButton">Show papers</button>
        </div>
        <div class="card" id="papers" style="display: none;">
            <br>
            <h3 style="margin-top: 10px; margin-bottom: 30px;">Papers</h3>
            <div class="links">
              <p><strong>ProtTrans: Towards Cracking the Language of Life's Code Through Self-Supervised Deep Learning and High Performance Computing</strong><br>Ahmed Elnaggar, Michael Heinzinger, Christian Dallago, Ghalia Rihawi, Yu Wang, Llion Jones, Tom Gibbs, Tamas Feher, Christoph Angerer, Martin Steinegger, Debsindhu Bhowmik, Burkhard Rost<br><em>arXiv</em>, July 2020 | <a href="https://arxiv.org/abs/2007.06225" target="_blank">Paper</a></p>
              <p><strong>BERTology Meets Biology: Interpreting Attention in Protein Language Models</strong><br>Jesse Vig, Ali Madani, Lav R. Varshney, Caiming Xiong, Richard Socher, Nazneen Fatema Rajani<br><em>arXiv</em>, July 2020 | <a href="https://arxiv.org/abs/2006.15222" target="_blank">Paper</a> | <a href="https://blog.einstein.ai/provis/" target="_blank">Blog</a></p>
              <p><strong>PEDL: extracting protein–protein associations using deep language models and distant supervision</strong><br>Leon Weber, Kirsten Thobe, Oscar Arturo Migueles Lozano, Jana Wolf, Ulf Leser<br><em>Bioinformatics</em>, July 2020 | <a href="https://doi.org/10.1093/bioinformatics/btaa430" target="_blank">Paper</a></p>
              <p><strong>Signal Peptides Generated by Attention-Based Neural Networks</strong><br>Zachary Wu, Kevin K. Yang, Michael J. Liszka, Alycia Lee, Alina Batzilla, David Wernick, David P. Weiner, Frances H. Arnold<br><em>ACS Synthetic Biology</em>, July 2020 | <a href="https://pubs.acs.org/doi/full/10.1021/acssynbio.0c00219#" target="_blank">Paper</a></p>
              <p><strong>USMPep: universal sequence models for major histocompatibility complex binding affinity prediction</strong><br>Johanna Vielhaben, Markus Wenzel, Wojciech Samek &amp; Nils Strodthoff<br><em>BMC Bioinformatics</em>, July 2020 | <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03631-1" target="_blank">Paper</a></p>
              <p><strong>Transforming the Language of Life: Transformer Neural Networks for Protein Prediction Tasks</strong><br>Ananthan Nambiar, Simon Liu, Mark Hopkins, Maeve Heflin,  View ORCID ProfileSergei Maslov, Anna Ritz<br><em>bioRxiv</em>, June 2020 | <a href="https://doi.org/10.1101/2020.06.15.153643" target="_blank">Paper</a></p>
              <p><strong>ProGen: Language Modeling for Protein Generation</strong><br>Ali Madani, Bryan McCann, Nikhil Naik, Nitish Shirish Keskar, Namrata Anand, Raphael R. Eguchi, Po-Ssu Huang, Richard Socher<br><em>bioRxiv</em>, March 2020 | <a href="https://doi.org/10.1101/2020.03.07.982272" target="_blank">Paper</a> | <a href="https://blog.einstein.ai/progen/" target="_blank">Blog</a></p>
              <p><strong>UDSMProt: universal deep sequence models for protein classification</strong><br>Nils Strodthoff, Patrick Wagner, Markus Wenzel, Wojciech Samek<br><em>Bioinformatics</em>, January 2020 | <a href="https://doi.org/10.1093/bioinformatics/btaa003" target="_blank">Paper</a>  </p>
              <p><strong>Modeling aspects of the language of life through transfer-learning protein sequences</strong><br>Michael Heinzinger, Ahmed Elnaggar, Yu Wang, Christian Dallago, Dmitrii Nechaev, Florian Matthes, Burkhard Rost<br><em>BMC Bioinformatics</em>, December 2019 | <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3220-8" target="_blank">Paper</a></p>
              <p><strong>Generative models for graph-based protein design</strong><br>John Ingraham, Vikas K. Garg, Regina Barzilay, Tommi Jaakkola<br><em>NeurIPS</em>, December 2019 | <a href="https://papers.nips.cc/paper/9711-generative-models-for-graph-based-protein-design.pdf" target="_blank">Paper</a></p>
              <p><strong>Accurate Protein Structure Prediction by Embeddings and Deep Learning Representations</strong><br>Iddo Drori, Darshan Thaker, Arjun Srivatsa, Daniel Jeong, Yueqi Wang, Linyong Nan, Fan Wu, Dimitri Leggas, Jinhao Lei, Weiyi Lu, Weilong Fu, Yuan Gao, Sashank Karri, Anand Kannan, Antonio Moretti, Mohammed AlQuraishi, Chen Keasar, Itsik Pe&#39;er<br><em>arXiv</em>, November 2019 | <a href="https://arxiv.org/abs/1911.05531" target="_blank">Paper</a></p>
              <p><strong>Athena: Automated Tuning of k-mer based Genomic Error Correction Algorithms using Language Models</strong><br>Mustafa Abdallah, Ashraf Mahgoub, Hany Ahmed, Somali Chaterji<br><em>Scientific Reports</em>, November 2019 | <a href="https://www.nature.com/articles/s41598-019-52196-4" target="_blank">Paper</a></p>
              <p><strong>Unified rational protein engineering with sequence-only deep representation learning</strong><br>Ethan C. Alley, Grigory Khimulya, Surojit Biswas, Mohammed AlQuraishi, George M. Church<br>Nature Methods, October 2019 | <a href="https://www.nature.com/articles/s41592-019-0598-1" target="_blank">Paper</a></p>
              <p><strong>Evaluating Protein Transfer Learning with TAPE</strong><br>Roshan Rao, Nicholas Bhattacharya, Neil Thomas, Yan Duan, Xi Chen, John Canny, Pieter Abbeel, Yun S. Song<br><em>bioRxiv</em>, June 2019 | <a href="https://doi.org/10.1101/676825" target="_blank">Paper</a> | <a href="https://bair.berkeley.edu/blog/2019/11/04/proteins/" target="_blank">Blog</a></p>
              <p><strong>Biological Structure and Function Emerge from Scaling Unsupervised Learning to 250 Million Protein Sequences</strong><br>Alexander Rives, Siddharth Goyal, Joshua Meier, Demi Guo, Myle Ott, C. Lawrence Zitnick, Jerry Ma, Rob Fergus<br><em>bioRxiv</em>, May 2019 | <a href="https://doi.org/10.1101/622803" target="_blank">Paper</a></p>
              <p><strong>A High Efficient Biological Language Model for Predicting Protein–Protein Interactions</strong><br>Yanbin Wang, Zhu-Hong You, Shan Yang, Xiao Li, Tong-Hai Jiang, Xi Zhou<br><em>Cells</em>, February 2019 | <a href="https://www.mdpi.com/2073-4409/8/2/122/htm" target="_blank">Paper</a></p>
              <p><strong>Learning continuous and data-driven molecular descriptors by translating equivalent chemical representations</strong><br>Robin Winter, Floriane Montanari, Frank Noé, Djork-Arné Clevert<br><em>Chemical Science</em>, November 2018 | <a href="https://doi.org/10.1039/C8SC04175J" target="_blank">Paper</a></p>
              <p><strong>Natural language processing in text mining for structural modeling of protein complexes</strong><br>Varsha D. Badal, Petras J. Kundrotas, Ilya A. Vakser<br><em>BMC Bioinformatics</em>, March 2018 | <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2079-4" target="_blank">Paper</a></p>
              <p><strong>Identifying the missing proteins in human proteome by biological language model</strong><br>Qiwen Dong, Kai Wang, Xuan Liu<br><em>BMC Systems Biology</em>, December 2016 | <a href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-016-0352-6" target="_blank">Paper</a></p>
              <p><strong>Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics</strong><br>Ehsaneddin Asgari, Mohammad R. K. Mofrad<br><em>PLOS ONE</em>, November 2015 | <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0141287" target="_blank">Paper</a></p>
              <p><strong>Survey of Natural Language Processing Techniques in Bioinformatics</strong><br>Zhiqiang Zeng, Hua Shi, Yun Wu, Zhiling Hong<br><em>Computational and Mathematical Methods in Medicine</em>, October 2015 | <a href="https://www.hindawi.com/journals/cmmm/2015/674296/" target="_blank">Paper</a></p>
          </div>
        </div>
        <div class="card">            
          <h2 style="font-family: arial;"><span id="technicalwriting"></span>Technical Writing</h3>
          <div class="links">
            <p id="para">Dynamic Replay of time-series data | <i>Utilizing matplotlib and double-ended queues in Python</i></p>
              <p style="font-size: 13px; margin-top: 1px;">April 6, 2020 · 2min read · Read article in <a href="https://towardsdatascience.com/dynamic-replay-of-time-series-data-819e27212b4b" target="_blank">Towards Data Science</a></p>
            <p id="para">Artefact Correction with ICA | <i>Illustrated with an example from the neurosciences</i><br></p>
              <p style="font-size: 13px; margin-top: 1px;">April 4, 2020 · 5min read · Read article in <a href="https://towardsdatascience.com/artefact-correction-with-ica-53afb63ad300" target="_blank">Towards Data Science</a></p>
          </div>
        </div>
        </div>
        <div class="right"></div>
    </div>
    <footer>
        <p style="color: white; margin: 0; position:absolute; top:50%; left:50%; -ms-transform: translate(-50%, -50%); transform: translate(-50%, -50%);">Thomas Dorfer | dorfer@molcure.io</p>
    </footer>
    <script>
      function responsiveness() {
        var x = document.getElementById("myTopnav");
        if (x.className === "nav") {
          x.className += " responsive";
        } else {
          x.className = "nav";
        }
      }

      function showPapers() {
        var x = document.getElementById("papers");
        var button = document.getElementById("myButton");
        if (x.style.display === "none") {
          x.style.display = "block";
          button.innerHTML = "Hide papers";
        } else {
          x.style.display = "none";
          button.innerHTML = "Show papers";
        }
      }
    </script>
</body>
</html>