<!DOCTYPE html>
<html lang="en-US">
<head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="css/styles.css">
    <link rel="shortcut icon" type="image/x-icon" href="logo.ico" />
    <title>Thomas Dorfer | NLProt</title>
</head>
<body>
    <ul class="nav">
        <li><a href="index.html">ABOUT ME</a></li>
        <li><a href="#">NLPROT</a></li>
        <li><a href="https://github.com/tadorfer" target="_blank">GITHUB</a></li>
        <li><a href="https://www.linkedin.com/in/thomasdorfer/" target="_blank">LINKEDIN</a></li>
        <li><a href="https://twitter.com/ThomasADorfer" target="_blank">TWITTER</a></li>
    </ul>
    <div class="column-header">
         <h1 id="header">Thomas Dorfer</h1>
         <h3 style="color: gray; text-align: center; font-size: 14px; font-weight: 100; padding: 0px 0px 85px 0px">DATA SCIENTIST | MOLCURE Inc.</h3>
    </div> 
    <div class="content">
        <div class="left"></div>
        <div class="midcolumn">
            <div class="card">
                <br><br><br><br<br><br><br><br>
                <h2>NLProt</h2>
                <p id="para">The application of Natural Language Processing (NLP) to protein sequences has recently gained traction in the field. There seem to be detectable
                    parallels between spoken language and protein sequences, with sentences being analogs of amino acid sequences, and words corresponding to individual amino acids.
                    This site is primarily for people interested in the application of NLP to proteins. The aim is to keep up-to-date with the recent literature in this field, so please
                    feel free to make a pull request if you have come across some relevant work or feel like I have missed something. 
                </p>
                <br><br>
            </div> 
            <div class="card" style="padding: 30px 0px;">
                <iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://plotly.com/~tadorfer/6.embed" height="600" width="100%"></iframe>
            </div>
            <div>
                <br><br>
                <h2>Papers</h2>
                <p><strong>BERTology Meets Biology: Interpreting Attention in Protein Language Models</strong><br>Jesse Vig, Ali Madani, Lav R. Varshney, Caiming Xiong, Richard Socher, Nazneen Fatema Rajani<br><em>arXiv</em>, July 2020 | <a href="https://arxiv.org/abs/2006.15222">Paper</a> | <a href="https://blog.einstein.ai/provis/">Blog</a></p>
                <p><strong>USMPep: universal sequence models for major histocompatibility complex binding affinity prediction</strong><br>Johanna Vielhaben, Markus Wenzel, Wojciech Samek &amp; Nils Strodthoff<br><em>Bioinformatics</em>, July 2020 | <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03631-1">Paper</a></p>
                <p><strong>Transforming the Language of Life: Transformer Neural Networks for Protein Prediction Tasks</strong><br>Ananthan Nambiar, Simon Liu, Mark Hopkins, Maeve Heflin,  View ORCID ProfileSergei Maslov, Anna Ritz<br><em>bioRxiv</em>, June 2020 | <a href="https://doi.org/10.1101/2020.06.15.153643">Paper</a></p>
                <p><strong>ProGen: Language Modeling for Protein Generation</strong><br>Ali Madani, Bryan McCann, Nikhil Naik, Nitish Shirish Keskar, Namrata Anand, Raphael R. Eguchi, Po-Ssu Huang, Richard Socher<br><em>bioRxiv</em>, March 2020 | <a href="https://doi.org/10.1101/2020.03.07.982272">Paper</a> | <a href="https://blog.einstein.ai/progen/">Blog</a></p>
                <p><strong>UDSMProt: universal deep sequence models for protein classification</strong><br>Nils Strodthoff, Patrick Wagner, Markus Wenzel, Wojciech Samek<br><em>Bioinformatics</em>, January 2020 | <a href="https://doi.org/10.1093/bioinformatics/btaa003">Paper</a>  </p>
                <p><strong>Modeling aspects of the language of life through transfer-learning protein sequences</strong><br>Michael Heinzinger, Ahmed Elnaggar, Yu Wang, Christian Dallago, Dmitrii Nechaev, Florian Matthes, Burkhard Rost<br><em>BMC Bioinformatics</em>, December 2019 | <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3220-8">Paper</a></p>
                <p><strong>Generative models for graph-based protein design</strong><br>John Ingraham, Vikas K. Garg, Regina Barzilay, Tommi Jaakkola<br><em>NeurIPS</em>, December 2019 | <a href="https://papers.nips.cc/paper/9711-generative-models-for-graph-based-protein-design.pdf">Paper</a></p>
                <p><strong>Athena: Automated Tuning of k-mer based Genomic Error Correction Algorithms using Language Models</strong><br>Mustafa Abdallah, Ashraf Mahgoub, Hany Ahmed, Somali Chaterji<br><em>Scientific Reports</em>, November 2019 | <a href="https://www.nature.com/articles/s41598-019-52196-4">Paper</a></p>
                <p><strong>Evaluating Protein Transfer Learning with TAPE</strong><br>Roshan Rao, Nicholas Bhattacharya, Neil Thomas, Yan Duan, Xi Chen, John Canny, Pieter Abbeel, Yun S. Song<br><em>bioRxiv</em>, June 2019 | <a href="https://doi.org/10.1101/676825">Paper</a> | <a href="https://bair.berkeley.edu/blog/2019/11/04/proteins/">Blog</a></p>
                <p><strong>Biological Structure and Function Emerge from Scaling Unsupervised Learning to 250 Million Protein Sequences</strong><br>Alexander Rives, Siddharth Goyal, Joshua Meier, Demi Guo, Myle Ott, C. Lawrence Zitnick, Jerry Ma, Rob Fergus<br><em>bioRxiv</em>, May 2019 | <a href="https://doi.org/10.1101/622803">Paper</a></p>
                <p><strong>A High Efficient Biological Language Model for Predicting Protein–Protein Interactions</strong><br>Yanbin Wang, Zhu-Hong You, Shan Yang, Xiao Li, Tong-Hai Jiang, Xi Zhou<br><em>Cells</em>, February 2019 | <a href="https://www.mdpi.com/2073-4409/8/2/122/htm">Paper</a></p>
                <p><strong>Learning continuous and data-driven molecular descriptors by translating equivalent chemical representations</strong><br>Robin Winter, Floriane Montanari, Frank Noé, Djork-Arné Clevert<br><em>Chemical Science</em>, November 2018 | <a href="https://doi.org/10.1039/C8SC04175J">Paper</a></p>
                <p><strong>Identifying the missing proteins in human proteome by biological language model</strong><br>Qiwen Dong, Kai Wang, Xuan Liu<br><em>BMC Systems Biology</em>, December 2016 | <a href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-016-0352-6">Paper</a></p>
                <p><strong>Enhanced protein domain discovery by using language modeling techniques from speech recognition</strong><br>Lachlan Coin, Alex Bateman, Richard Durbin<br><em>PNAS</em>, April 2003 | <a href="https://www.pnas.org/content/100/8/4516">Paper</a></p>
            </div>
        </div>
        <div class="right"></div>
    </div>
    <footer>
        <p style="margin: 0; position:absolute; top:50%; left:50%; -ms-transform: translate(-50%, -50%); transform: translate(-50%, -50%);">Thomas Dorfer | dorfer@molcure.io</p>
    </footer>
</body>
</html>